keps/sig-node/2400-node-swap/README.md

```diff
@@ -441,6 +472,113 @@ Additionally, end user may decide to disable swap completely for a Pod or a cont
+(see the [summary](#summary) section above for more info).
+
+##### Brief technical overview of swap and evictions
+
+With swap being disabled, the memory eviction threshold should be set in a way that will let kubelet's
+eviction manager identify memory spikes in time so it can execute custom logic and take care of reclaiming
+memory and evicting pods before the kernel's OOM killer is invoked.
+
+However, with swap enabled, the situation is fundamentally different.
+In order for the kernel to start swapping memory, either the pod must breach its memory limits
+(that are not relevant for kubelet-level evictions) or the node must use all of its physical memory.
+Since swapping is a heavy operation in terms of performance, the kernel will try avoiding
+it as long as it can, and will start swapping only when it lacks physical memory.
+
+Therefore, when swap is enabled, it is acceptable to use all the available RAM which will result in the node
+using some of the swap memory.
+
+##### Current eviction limitations
+
+Because evictions are out-of-scope for this KEP, the eviction manager would remain swap-unaware.
+
+Let's say that on a specific node, the kernel would start swapping when there is less than X free physical memory
+available (see more detailed explanation [below](#advanced-best-practices-for-manually-setting-memory-evictions)).
+This means either one of the two behaviors will take place:
+1. Memory evictions threshold > X: In this scenario, kubelet's eviction manager would kick in before the kernel has
+   a chance to swap memory.
+   This means that the kernel would generally not be able to swap memory (unless the memory spike
+   is fast enough for the eviction manager to handle).
+1. Memory evictions threshold <= X: In this scenario, the kernel would start swapping memory before the memory threshold 
+   would be met. This practically means that the eviction manager would never kick in, hence it is turned off (unless
+   for extreme cases where the memory spike is faster than the kernel's ability to swap memory).
+   If the cluster admin sees this as a desirable state, the `--experimental-allocatable-ignore-eviction` kubelet flag
+   can be used in order to emphasize that memory evictions are indeed turned off. 
+
+The cluster admin can choose which of the above is the desired behavior.
+If the first is applied, kubelet would be able to evict pods but pods would be able to swap only if memory limits
+are breached.
+If the second is applied, kubelet evictions would be practically disabled but node-level pressure would result in
+swapping.
+
+Advanced cluster admins can use the approach outlined [below](#advanced-best-practices-for-manually-setting-memory-evictions).
+
+##### Advanced best-practices for manually setting memory evictions
+
+Let's simplistically explain how the kernel decides to start swapping.
+The kernel uses three "watermarks" (i.e. thresholds), which are called "high", "min", and "low", such that high > min > low.
```

Comment:
typo?
- `high > low > min`

Reply:
It isn't a typo, although I see how it can be a bit confusing.
These values represent the amount of free memory **left**, so a "high" amount of memory left is larger than a low amount of memory.

As written in the KEP itself:
```
This is a real example from a 64GB RAM machine with the default kernel configurations. On such a machine:
* The `high` watermark is at `187.32Mi`.
* The `low` watermark is at `125.54Mi`.
* The `min` watermark is at `63.76Mi`.
```

Reply:
Your example seems to be 
- `low > min`

Reply:
Ah, you're right!
Thank you @pacoxu! Great catch! fixed